python/kserve/kserve/protocol/infer_type.py

```diff
@@ -791,8 +880,14 @@ def to_grpc(self) -> ModelInferResponse:
         """
         infer_outputs = []
         raw_output_contents = []
+        use_raw_outputs = False
+        # If FP16 datatype is present in the outputs use raw outputs.
+        if _contains_fp16_datatype(self):
+            use_raw_outputs = True
```

Comment:
What happens if only one of the outputs use FP16?

Reply:
The response will be returned as raw outputs

Reply:
same for output according to 
https://github.com/kserve/open-inference-protocol/blob/52528cf4c48368114874e99217d25b9e006e494a/specification/protocol/open_inference_grpc.proto#L257

Reply:
I mean the all the outputs will be sent as bytes which will be stored in raw_ouput_contents field. The contents field will be empyt.